tasks/section5.yml

```diff
@@ -231,15 +345,19 @@
       - rule_5.2.4
 
 - name: "SCORED | 5.2.4 | PATCH | Ensure SSH X11 forwarding is disabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^(#)?X11Forwarding \d'
+      line: 'X11Forwarding no'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.4
 
 - name: "SCORED | 5.2.5 | AUDIT | Ensure SSH MaxAuthTries is set to 4 or less"
-  command: /bin/true
+  shell: grep "^MaxAuthTries 4" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*